tests/auto/tst_declarativetabmodel/tst_declarativetabmodel.cpp

```diff
@@ -283,9 +281,6 @@ void tst_declarativetabmodel::multipleTabsWithSameUrls()
     QCOMPARE(tabModel->data(modelIndex, DeclarativeTabModel::UrlRole).toString(), page2Tab1Url);
     QVERIFY(tabModel->data(modelIndex, DeclarativeTabModel::TitleRole).toString().isEmpty());
 
-    tabModel->updateTitle(tab2, true, page2Tab2Url, page2Tab2Title);
-    QCOMPARE(tabModel->activeTab().title(), page2Tab2Title);
-
```

Comment:
Yes. I'd like to rewrite tst_declarativetabmodel where each test case would run in the same environment set up in setUp() and tearDown() functions.